--- a/package/kernel/qca-ssdk/Makefile
+++ b/package/kernel/qca-ssdk/Makefile
@@ -81,6 +81,7 @@ define Build/Compile
 	+$(MAKE) $(PKG_JOBS) $(MAKE_FLAGS) -C $(PKG_BUILD_DIR)/mk $(LNX_CONFIG_OPTS)
 	$(INSTALL_DIR) $(PKG_SYMVERS_DIR)
 	$(CP) $(QCA_SSDK_SYMVERS) $(PKG_SYMVERS_DIR)/$(PKG_NAME).symvers
+	python3 $(TOPDIR)/scripts/patch_modinfo.py $(PKG_BUILD_DIR)/build/bin/qca-ssdk.ko mdio-bitbang mdio-i2c
 endef
 
 define Build/InstallDev
